<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassChangeDialog</class>
 <widget class="QDialog" name="PassChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение пароля</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Старый пароль</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="eOld">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Новый пароль</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="eNew">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Повторите ввод</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="eConfirm">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="bOk">
     <property name="text">
      <string>ОК</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="bCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bOk</sender>
   <signal>clicked()</signal>
   <receiver>PassChangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>PassChangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
